src/main/drivers/pwm_output.c

```diff
@@ -352,6 +353,28 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         }
     }
 }
+
+void pwmWriteDshotCommandAll(uint8_t motorCount, uint8_t command)
+{
+    if (command <= DSHOT_MAX_COMMAND) {
+        for (uint8_t index = 0; index < motorCount; index++) {
+
+            unsigned repeats;
+            if ((command >= 7 && command <= 10) || command == 12 || (command >= 20 && command <= 21)) {
+                repeats = 10;
+            }
+            else {
+                repeats = 1;
+            }
+            for (; repeats; repeats--) {
+                pwmWritePtr(index, command);
```

Comment:
You don't set `motor->requestTelemetry`. The commands sent will just be ignored.